### DISCUSSION TITLE
Can’t star while clicking star button.

### DISCUSSION THREAD

**Discussion by rikuesaria** [community/community#51109] [Mobile]
It shows with following report and the star button turns grey. Tried a lot but not solved.

```
Something went wrong while executing your query. Please include
E9D2:6EF2:90FA:98C3:64218C77' when reporting this issue.
```

![image](https://user-images.githubusercontent.com/50106552/227942732-5d424539-cff3-4beb-a156-d35e4f5b900f.jpeg)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.105.0 (94356885)
Device: iPad Mini 6 (iOS 16.3)
</details

**Comment by GyulyVGC** [comment]
It's a problem on GitHub side: see [here](https://www.githubstatus.com) @rikuesaria
We just have to wait and hopefully it'll be solved soon.
Don't forget to mark this comment as answer so that other people can view this discussion as solved! 

![Screenshot 2023-03-27 at 15 05 15](https://user-images.githubusercontent.com/100347457/227947004-184645b4-1590-40d8-afe0-825a878b6934.png)

**Comment by PajaritoMoyqi** [comment]
Now maybe it works!

**Comment by iskae** [comment]
Hello @rikuesaria 👋 

Thanks a lot for reaching out ❤️ 

Unfortunately I was not able to reproduce the issue 😅 Is it still reproducible on your side? If so, would you be able to record a small video of the problem that you are experiencing?

Cheers!
